store/postgres/migrations/000001_create_tags_templates_fields_table.down.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+DROP TABLE IF EXISTS tags;
+
+DROP TABLE IF EXISTS fields;
+
+DROP TABLE IF EXISTS templates;
+
```

Comment:
shouldn't we drop the indexes as well?

Reply:
AFAIK all constraints (including indices) are also get removed once we drop the table.